site/sfguides/src/terraforming_snowflake/terraforming_snowflake.md

```diff
@@ -244,40 +241,35 @@ Duration: 5
 
 All databases need a schema to store tables, so we'll add that and a service user so that our application/client can connect to the database and schema. The syntax is very similar to the database and warehouse you already created. By now you have learned everything you need to know to create and update resources in Snowflake. We'll also add privileges so the service role/user can use the database and schema.
 
-You'll see that most of this is what you would expect. The only complicated part is creating the private key. Because the Terraform TLS private key generator doesn't export the public key in a format that the Terraform provider can consume, some string manipulations are needed.
+You'll see that most of this is what you would expect. The only new part is creating the private key. Because the Terraform TLS private key generator doesn't export the public key in a format that the Terraform provider can consume, some string manipulations are needed.
 
 1. Change the warehouse size in the `main.tf` file from `xsmall` to `small`.
 
-1. Add the following resources to your `main.tf` file:
+2. Add the following resources to your `main.tf` file:
 
-    ```JSON
+    ```
         provider "snowflake" {
-            alias = "security_admin"
             role  = "SECURITYADMIN"
         }
 
         resource "snowflake_role" "role" {
-            provider = snowflake.security_admin
             name     = "TF_DEMO_SVC_ROLE"
         }
 
         resource "snowflake_database_grant" "grant" {
-            provider          = snowflake.security_admin
```

Comment:
this is important as there are 2 different roles being used in this project.

Reply:
fixed. the system admin is considered the default, and this one will be the non-default. that way not every resource needs their provider explicitly defined. a better way to do this would be to shove all the resources using one provider into a module, and all the resources using the other provider into a different module.